packages/clients/meldemichel/src/language.ts

```diff
@@ -121,6 +129,21 @@ const language: LanguageOption[] = [
               'Die Verortung ist fehlgeschlagen. Bitte benutzen Sie Karte und Suche, um einen Schaden innerhalb von Hamburg zu melden.',
           },
         },
+        gfi: {
+          list: {
+            header: 'Meldungsliste',
```

Comment:
https://github.com/Dataport/polar/pull/30/commits/ed7d60be1133bc2edc7020c30a57533fe0b09ab6 I went in with an axe and removed some obvious ones. Looking at it, I don't think we'll use that many of the remaining anyway. If you agree, I'd just delete the rest now, too, since we can always take inspiration from the Prodmichel anyway.

Reply:
Deleting them now seems like the best approach 👍